plugins/serializers/template/template_test.go

```diff
@@ -0,0 +1,131 @@
+package template
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+)
+
+var basicMetric = metric.New(
+	"cpu",
+	map[string]string{
+		"host": "localhost",
+		"cpu":  "CPU0",
+	},
+	map[string]interface{}{
+		"x": 42.0,
+		"y": 43.0,
+	},
+	time.Unix(100, 0),
+)
```

Comment:
Please also (re-)define this in the tests _or_ put this into a file and use "testcase" based testing as [in the lookup processor](https://github.com/influxdata/telegraf/blob/54b1009a9f0a2232e1c1b8456df48c614251f1ec/plugins/processors/lookup/lookup_test.go#L42)...

Reply:
Done.